src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write your code here
+const list = document.createElement('ul');
+
+document.body.append(list);
+
+const request = (
+  endPoint = '',
+  dataFormat = '.json',
+  params = { method: 'GET' },
+  baseUrl = 'https://mate-academy.github.io/phone-catalogue-static/api'
+) => {
```

Comment:
```suggestion
const request = (endPoint) => {
```
В цій задачі, по суті, ти скрізь передаєш лише ендпоінти. Можна прибрати всі аргументи, які ніде не використовуються більше, і просписати їх явно прямо всередині методу fetch. А для baseUrl просто створити окрему змінну